### ISSUE TITLE
Documentation of examples feature request: Clearly delineate when text names are labels or handles.

### ISSUE THREAD

**Issue by cheako** [corrosion-rs/corrosion#340]
For example, can I change `cxxbridge-cpp`?  I suggest appending some Elf's name, to use `cxxbridge-cpp-jorty` instead.  This offers some benefits, like uniquely identifying where text must match and the clarity that this can be altered so for example there could be more than one.

My A problem is that this project has multiple binaries and I'm thinking I'll want multiple crates in the future.

**Comment by jschwe** [comment]
>For example, can I change cxxbridge-cpp

Yes, that name was arbitrary. Please do note that the cxxbridge integration is still considered experimental, and is not yet available in any official release of corrosion and thus also not documented.

**Comment by cheako** [comment]
Yeah, after I opened this I saw that I was looking at test cases and not examples.  I think cxx2rust and rust2cxx names are backwards in some cases, should the lang containing `main()` not be the originator and listed first?
